agent/runtime-attach/build.gradle.kts

```diff
@@ -0,0 +1,11 @@
+plugins {
+  id("ai.java-conventions")
+}
+
+dependencies {
+  implementation("io.opentelemetry.contrib:opentelemetry-runtime-attach:1.14.0-alpha")
+  implementation("com.microsoft.azure:applicationinsights-agent:" + "3.2.11")
+  testImplementation("io.opentelemetry:opentelemetry-extension-annotations:1.14.0")
```

Comment:
to avoid hard-coding this version, check out https://github.com/microsoft/ApplicationInsights-Java/blob/7eb023e32f5ec4987c09f618f326ff92baf41815/agent/instrumentation/applicationinsights-web-2.3/build.gradle.kts#L20-L25

Reply:
Code updated Please [recheck](https://github.com/microsoft/ApplicationInsights-Java/blob/runtime-attach/agent/runtime-attach/build.gradle.kts)